runs/send_debug.sh

```diff
@@ -2,6 +2,15 @@
 OPENWBBASEDIR=$(cd "$(dirname "$0")/../" && pwd)
 RAMDISKDIR="${OPENWBBASEDIR}/ramdisk"
 
+
+function merge_log_files(){
+	if [[ -f "${RAMDISKDIR}/${1}.log.1" ]]; then
+		cat "${RAMDISKDIR}/${1}.log.1" "${RAMDISKDIR}/${1}.log" | tail -n $2
+	else
+		tail -$2 "${RAMDISKDIR}/${1}.log"
+	fi
+}
```

Comment:
Was ist mit Protokollen, die nicht in der Ramdisk liegen? Der Pfad sollte optional überschreibbar sein.
Dann sollte in Zeile 8 und 10 die gleiche Syntax für `tail` verwendet werden.
```suggestion
function merge_log_files(){
	LOGPATH=${3:-$RAMDISKDIR}
	if [[ -f "${LOGPATH}/${1}.log.1" ]]; then
		cat "${LOGPATH}/${1}.log.1" "${LOGPATH}/${1}.log" | tail -n $2
	else
		tail -n $2 "${LOGPATH}/${1}.log"
	fi
}
```
Jetzt kann mit einem dritten Parameter ein Pfad angegeben werden. Wird das weggelassen, wird der Inhalt von `$RAMDISKDIR` verwendet.